app/kafka/manager/BrokerViewCacheActor.scala

```diff
@@ -19,6 +19,8 @@ class BrokerViewCacheActor(kafkaStateActorPath: ActorPath, clusterConfig: Cluste
 
   private[this] var brokerTopicPartitions : Map[Int, BVView] = Map.empty
 
+  private[this] var topicIdentities : IndexedSeq[TopicIdentity] = IndexedSeq.empty
```

Comment:
Based on the comment below about IndexedSeq[(String, Option[TopicIdentity])], you could store a map here, Map[String, TopicIdentity] where map is keyed on topic name.